src/semantic.rs

```diff
@@ -152,6 +152,8 @@ pub fn new_state(ast: Box<Root>) -> ProgramState {
 }
 
 impl ProgramState {
+    pub const IR_OUTPUT_FILENAME: &'static str = "out.ir";
```

Comment:
idk if this is how you want it to be but I wanted it to be symmetric to the C output